packages/accordions/examples/advanced.md

```diff
@@ -0,0 +1,205 @@
+### Accordion
+
+The Accordion component is implemented using the [W3C Accordion pattern](https://www.w3.org/TR/wai-aria-practices/#accordion).
+
+#### With custom header elements
+
+This advanced example demonstrates additional header text and menu buttons in the `Accordion.Header`.
+
+```jsx
+const { Well } = require('@zendeskgarden/react-notifications/src');
+const { Toggle, Field, Label } = require('@zendeskgarden/react-forms/src');
+const { IconButton } = require('@zendeskgarden/react-buttons/src');
+const GearIcon = require('@zendeskgarden/svg-icons/src/16/gear-stroke.svg').default;
+const FolderIcon = require('@zendeskgarden/svg-icons/src/16/folder-open-stroke.svg').default;
+const { Dropdown, Menu, Item, Trigger } = require('@zendeskgarden/react-dropdowns/src');
+
+const StyledHint = styled.span`
+  font-size: ${props => props.theme.fontSizes.sm};
+  font-weight: normal;
+  font-style: italic;
+`;
+
+const StyledButtonGroup = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  width: ${props => props.theme.space.base * 28}px;
+`;
+
+const AdvancedExample = () => {
+  const [isCollapsible, setIsCollapsible] = React.useState(false);
+  const [isExpandable, setIsExpandable] = React.useState(false);
+  const [isBare, setIsBare] = React.useState(false);
+  const [isCompact, setIsCompact] = React.useState(false);
+
+  const buttonGroup = (
+    <StyledButtonGroup>
+      <Dropdown onSelect={item => alert(item)}>
+        <Trigger>
+          <IconButton aria-label="Settings" title="Settings">
+            <GearIcon />
+          </IconButton>
+        </Trigger>
+        <Menu placement="top" hasArrow>
+          <Item value="option-1">Option 1</Item>
+          <Item value="option-2">Option 2</Item>
+          <Item value="option-3">Option 3</Item>
+        </Menu>
+      </Dropdown>
+      <Dropdown onSelect={item => alert(item)}>
+        <Trigger>
+          <IconButton aria-label="Settings" title="Settings">
+            <FolderIcon />
+          </IconButton>
+        </Trigger>
+        <Menu placement="top" hasArrow>
+          <Item value="option-1">Option 1</Item>
+          <Item value="option-2">Option 2</Item>
+          <Item value="option-3">Option 3</Item>
+        </Menu>
+      </Dropdown>
+    </StyledButtonGroup>
```

Comment:
I think the advanced example leaves out the `isCompact` toggle because of the spacing limitations it causes for the menu buttons. If a consumer wants to add Garden menu buttons in the header, the `isCompact` prop should not be used as doing so constrains the space. 

IMO, the `isCompact` prop is somewhat inflexible, and I was hesitant about adding this prop to the `Accordion`. I think it makes the most sense for the `Accordion` to come with default padding. If consumers decide to render large menu buttons, they can add more padding. If they want less padding, they can customize that too.

@steue what do you think? How should the menu buttons respond to `isCompact={true}`?

Reply:
@hzhu, the system should work together here. If it doesn't, then compact sizing would need to be re-thought. I would expect you to use `size="small"` icon buttons.

Reply:
@hzhu @jzempel it makes sense to me that icon buttons scale with `isCompact`

there's [a different thread here](https://github.com/zendeskgarden/react-components/pull/725#discussion_r414067549) about `isCompact` padding. I believe this can be fixed by changing the vertical padding so there is `8px` of spacing between icon and the header container. [Visual explorations of icon button scaling + spacing on Abstract.](https://share.goabstract.com/eee22b64-e2ff-4bed-8e7f-6588b57a4c13?collectionLayerId=79334da2-e6e6-47a9-99cb-38d26a3292da&mode=design)

Reply:
On these pages, we need to stress test all combinations. We'll reserve the `website` (needed so badly) for guard-railing expected usage. Please add an `isCompact` toggle here.

Reply:
The `isCompact` toggle has been added.